ResearchKit/Common/ORKTaskViewController.m

```diff
@@ -910,10 +912,14 @@ - (void)showViewController:(ORKStepViewController *)viewController goForward:(BO
     
     NSString *progressLabel = nil;
     if ([self shouldDisplayProgressLabel]) {
-        ORKTaskProgress progress = [_task progressOfCurrentStep:viewController.step withResult:[self result]];
-        
-        if (progress.total > 0) {
-            progressLabel = [NSString stringWithFormat:ORKLocalizedString(@"STEP_PROGRESS_FORMAT", nil) ,ORKLocalizedStringFromNumber(@(progress.current + 1)), ORKLocalizedStringFromNumber(@(progress.total))];
+        if (viewController.parentReviewStep && viewController.parentReviewStep.isStandalone) {
+            progressLabel = viewController.parentReviewStep.title;
```

Comment:
Okay I understand, say we add a `hideProgressLabel` property on `ORKStepViewController`, is this your intention: `willReviewStep:` sets this property to `YES` before calling `showViewController:`. If set to `YES`, `collectToolbarItems` should pick up the title (since `shouldDisplayProgressLabel` returns `NO`)?